packages/cyberstorm/src/components/Button.tsx

```diff
@@ -19,11 +24,19 @@ export interface ButtonProps {
 /**
  * Cyberstorm Button component
  */
-export const Button: React.FC<ButtonProps> = (props) => {
-  const { label, leftIcon, rightIcon, colorScheme, onClick } = props;
+export const Button: React.FC<ButtonProps> = React.forwardRef<
+  HTMLButtonElement,
+  ButtonProps
+>((props: PropsWithChildren<ButtonProps>, forwardedRef) => {
+  const { label, leftIcon, rightIcon, colorScheme, onClick, ...rest } = props;
```

Comment:
Continuing the `...rest` discussion: the name of the variable doesn't convey much information about the reasons it's used, as I think is proven by me needing to ask what it's for 😅 . Since the main point is for passing Radix props, and at the same time React.FC props, would something like `radixFCProps` make sense? Or `forwardedProps` to be generic?

Reply:
I like forwardedProps 👍